src/Components/ApiFetch/ApodApiFetch.tsx

```diff
@@ -0,0 +1,24 @@
+﻿import React from "react";
+
+
+
+export function parseApiData() {
+    var requestOptions: any = {
+        method: 'GET',
+        redirect: 'follow'
+    };
```

Comment:
We can probably get away without these - we just need the default values, so its fine to just skip this whole thing!